Merge from dev to master

omcdanie opened the pull request:
Make smaller commits more often.

omcdanie committed:
Minor error fixing. Create new branch.

omcdanie committed:
Create separate components for finished and current bucket items.  Create initial text-area component.

omcdanie committed:
TextArea can be built with rows and cols or through styling the html component directly.  Add text area to mini bio section.

omcdanie committed:
Code cleanup

omcdanie committed:
Further css updates and mini-refinements.

omcdanie committed:
Add scroll functionality to buckets in profile.

omcdanie committed:
Shorten the height of the bucket scrolling.

djpara committed:
Merge pull request #12 from BeforeTheEnd-Inc/ProfileUIEnhancements

Profile ui enhancements

omcdanie committed:
Change Current bucket scroll to be dynamic.  It will show 3 items at the most.  Add link to add/create new bucket item.

omcdanie committed:
fix formatting

djpara committed:
Merge pull request #13 from BeforeTheEnd-Inc/ProfileUIEnhancements

Profile ui enhancements

omcdanie committed:
Update profile links. Add new font to the whole application.  This can and should be refined as necessary.

Commit:
mini bio ui mostly done

Commit:
Merge remote-tracking branch 'remotes/origin/ProfileUIEnhancements' into mini_bio

djpara committed:
Merge pull request #15 from BeforeTheEnd-Inc/ProfileUIEnhancements

Minor updates

omcdanie committed:
Merge pull request #16 from BeforeTheEnd-Inc/mini_bio

Mini bio

alredi commented on client/IndexComponent.js:
```diff
@@ -11,6 +13,7 @@ export default class Index extends Component {
             <BrowserRouter>
                 <Switch>
                     <Route exact path='/' component={ProfileView} />
+                    {/*<Route exact path='/' component={Bucket} />*/}
```
We shouldn't be pushing commented out code

alredi commented on client/IndexComponent.js:
```diff
@@ -2,6 +2,8 @@ import React, {Component} from 'react';
 import {Route, BrowserRouter, Switch} from 'react-router-dom';
 
 import Login from '../imports/components/LoginComponent';
+import Bucket from '../imports/ui/buckets/BucketView';
```
Let's try to be consistent - here we're importing "Bucket" and below "ProfileView", where the original files both have "View" in the title.

omcdanie replied:
I hear what you're saying here, but keep in mind these imports are used for testing components until routing is implemented.

alredi commented on client/Main.css:
```diff
@@ -1,21 +1,27 @@
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
+
 html,
```
This shows how to apply a global font so it doesn't need repeated for every item:

https://stackoverflow.com/questions/7025756/how-to-apply-global-font-to-whole-html-document

alredi commented on imports/ui/profile/BiographyComponent.js:
```diff
@@ -0,0 +1,96 @@
+                        this component that will allow the user to
+                        update and change her/his biography at any time.
+                    </p>
+                    {/*<Label*/}
```
More commented out code - this shouldn't be pushed.

alredi reviewed (approved):
made a few comments, but none of them are major.

Commit:
implemented radio buttons

omcdanie committed:
Merge pull request #17 from BeforeTheEnd-Inc/mini_bio

implemented radio buttons

allisondiller committed:
Navbar complete with routing.

NOTE: This fucks up CSS due to importing bootstrap. Will put in bug to fix CSS in future sprint.

allisondiller committed:
Initial navbar

Commit:
added command to bucketful-build.sh to remove images with a "<none>" tag

Commit:
added if statment to check for "<none>"/untagged image before running docker rmi. This prevents the following error "docker rmi requires at least 1 argument."

allisondiller committed:
Merge remote-tracking branch 'origin/navbar' into navbar

# Conflicts:
#	imports/components/MenuComponent.js

omcdanie committed:
Initial bucket implementation.  Needs substantial work.

omcdanie committed:
Merge pull request #19 from BeforeTheEnd-Inc/navbar

This will need some fixing.

allisondiller committed:
Saved react-router

allisondiller committed:
Saved react-router

allisondiller committed:
Merge remote-tracking branch 'origin/router' into router

omcdanie committed:
Merge pull request #20 from BeforeTheEnd-Inc/clean_old_docker_images

Clean old docker images

omcdanie committed:
Merge pull request #21 from BeforeTheEnd-Inc/router

router

allisondiller committed:
Provided nav for "Home"

djpara committed:
Merge pull request #22 from BeforeTheEnd-Inc/router

router

Commit:
created sign up ui

Commit:
added submit button

Commit:
replaced reactstrap with react-bootstrap

Commit:
Minor refactor and added css

Commit:
added profile schema

Commit:
increased menu items font size

Commit:
update sign up ui and implemented basic form functionality

Commit:
profiles - rewrote profiles.js and not have persistent storing using simple schemas

alredi committed:
Merge pull request #23 from BeforeTheEnd-Inc/signup_ui

created sign up ui

omcdanie committed:
Add preliminary bucket page. Needs some css love (along with the rest of the application).

omcdanie committed:
Add bucket changes...

omcdanie committed:
Remove unused code.

alredi committed:
Merge pull request #24 from BeforeTheEnd-Inc/bucket

Bucket

allisondiller committed:
Add bucket front end

allisondiller committed:
removed commented out code

alredi committed:
Merge branch 'dev' into addbucket

omcdanie committed:
Merge pull request #25 from BeforeTheEnd-Inc/addbucket

Add bucket front end

omcdanie committed:
Update css.

alredi committed:
Merge pull request #26 from BeforeTheEnd-Inc/bucket

Update css.
